x-pack/test/cloud_security_posture_functional/page_objects/csp_dashboard_page.ts

```diff
@@ -82,6 +83,35 @@ export function CspDashboardPageProvider({ getService, getPageObjects }: FtrProv
       }
     },
 
+    getAllComplianceScoresByCisSection: async (tab: DashboardTabs) => {
+      await dashboard.getDashoard(tab);
+      const pageContainer = await testSubjects.find('pageContainer');
+      return await pageContainer.findAllByTestSubject('cloudSecurityFindingsComplianceScore');
+    },
+
+    getDashoard: async (tab: DashboardTabs) => {
+      if (tab === 'Cloud') {
+        return await dashboard.getCloudDashboard();
+      }
+      if (tab === 'Kubernetes') {
+        return await dashboard.getKubernetesDashboard();
+      }
+    },
+
+    getFindingsLinkAtIndex: async (tab: DashboardTabs, linkIndex = 0) => {
+      await dashboard.getDashoard(tab);
+      const pageContainer = await testSubjects.find('pageContainer');
+      const allLinks = await pageContainer.findAllByXpath(`//button[contains(@class, 'euiLink')]`);
```

Comment:
if possible it's better to add a test subject. Who knows what other links might be added there with time

Reply:
Added a new [task](https://github.com/elastic/security-team/issues/9917) to implement test subjects